core/trino-main/src/main/java/io/trino/operator/DeleteAndInsertMergeProcessor.java

```diff
@@ -187,13 +189,14 @@ private void addInsertRow(PageBuilder pageBuilder, ColumnarRow mergeCaseBlock, i
         }
 
         // Add the operation column == insert
-        TINYINT.writeLong(pageBuilder.getBlockBuilder(dataColumnChannels.size()), INSERT_OPERATION_NUMBER);
+        int dataColumnChannelsSize = dataColumnChannels.size();
```

Comment:
var extraction is a redundant change, either revert (preferred, for simplicity), or separate